algorithms/JavaScript/src/maths/fibonacci-series.js

```diff
@@ -0,0 +1,23 @@
+// algorithm to generate first n numbers of fibonacci series
+
+function fibonacci(n) {
+  // Initialize array 'fibS' with the first two numbers of the fibonacci series
+  // [0, 1]
+  const fibS = [0, 1];
```

Comment:
Rename the parameter to `series`
```suggestion
  const series = [0, 1];
```

Reply:
done ✔️  
Please could you explain to me why that was needed? 
Thank you